portfolio1.css

```diff
@@ -0,0 +1,142 @@
+body {
+	background-color: rgb(165,42,42);
+	color: white;
+}
+
+a {
+	color: red;
+}
+
+.top-bar {
+	display: flex;
+	justify-content: space-between;
+}
+
+.logo-link {
+	width:15%;
+	height: 85%;
+}
+
+.accessibility-link {
+	background-color: rgb(75,0,130);
+	height: 15%;
+}
+.form {
+	display: flex;
+	flex-direction: column;
+}
+
+button {
+	 color:	rgb(128,0,128);
+	 background-color: rgb(255,0,0);
+	 width: 12%;
+	 font-family: Helvetica, Arial, Times;
+	 font-weight: bold;
+}
+.buttons1 {
+     padding: 1%;
+     display: flex;
+	 justify-content: space-between;
+}
+
+#submit {
+	color: white;
+ font-weight: bold;
+ background-color: purple;
+}
+
+.main-image-div{
+	position: relative;
+}
+
+.main-image {
+	width: 100%;
+}
+
+.main-image-heading {
+	position: absolute;
+	top: 65%;
+}
+
+.text-box-1 {
+	margin: 10px 20px 10px 20px;
+}
+
+h2 {
+	margin: 5px;
+}
+h3 {
+	margin: 5px;
+}
+h4 {
+	margin: 5px;
+}
+
+.main-body{
+	display: flex;
```

Comment:
Hi, I dont get why you put display flex, but nothing else...

Reply:
Ah, yes. I wanted the main body to be flex because the default flex-direction is row, so when I place the child elements main-text and images inside it they will be organised in a row, despite each of THEIR elements displaying as columns. Does that make sense?